oak_functions_containers_app/src/lib.rs

```diff
@@ -217,7 +217,9 @@ where
 
         let instance = self.get_instance()?;
         while let Some(chunk) = request.next().await {
-            instance.extend_lookup_data_chunk(chunk?);
+            instance
+                .extend_lookup_data_chunk(chunk?)
+                .map_err(|e| tonic::Status::failed_precondition(e.to_string()))?;
```

Comment:
We have a `map_status` helper function for this; look at the `extend_next_lookup_data` function for an example.

Reply:
Done.